test/dynamicalsystems.jl

```diff
@@ -124,4 +124,16 @@ dict_keys = ["Sine wave","White noise","Hénon (chaotic)","Hénon (periodic)"]
     @test amat == Matrix(rmat) - I
     rna_dict = rna(rmat)
     rna_dict[:density] == recurrencerate(rmat)
+
+    #test sparse matrices
+    rqapar_sparse = rqa(rmat.data, theiler=1, lmin=3, border=20)
+    rqadiag_sparse = rqa(rmat.data, theiler=1, lmin=3, border=20, onlydiagonal=true)
+    for p in keys(rqadiag_sparse)
+        @test rqapar_sparse[p]==rqadiag_sparse[p]
+    end
+
+    #check for erroring here with sparse matrices
+    @windowed recurrencerate(rmat.data,theiler=1) width=50 step=40
+    @windowed rqa(rmat.data,theiler=1) width=50 step =40
+    coordinates(rmat.data)
```

Comment:
there is no `@test` done here.

Reply:
I was simply checking if it were to error out similar to  what is now line 113 in this file. I have added a test for coordinates though. Considering `rmat` has not changed, I figured you mostly just needed to check it did not error out and I did test that the values coming out would be the same since I have already added tests that the `rqa` functions work with sparse arrays, so just need to check that the `@windowed` macro worked.

I went ahead and added two `@test` though for `coordinates` and the `:RR` computed in 136 and 137.